Pod/Classes/VOKCoreDataManager.m

```diff
@@ -521,6 +499,38 @@ + (void)writeToTemporaryContext:(void (^)(NSManagedObjectContext *tempContext))w
     }];
 }
 
++ (void)importArrayInBackground:(NSArray *)inputArray
+                       forClass:(Class)objectClass
+                     completion:(VOKObjectIDReturnBlock)completion
+{
+    [[VOKCoreDataManager sharedInstance] managedObjectContext];
```

Comment:
I'm assuming this is just a poke to make sure the MOC has been setup, right? If so, could you include a comment?